base/BUILD.gn

```diff
@@ -2969,34 +2973,39 @@ source_set("base_stack_sampling_profiler_test_util") {
   }
 }
 
-if (is_apple) {
-  bundle_data("base_unittests_bundle_data") {
-    testonly = true
-    sources = [
-      "//tools/metrics/histograms/enums.xml",
-      "test/data/file_util/binary_file.bin",
-      "test/data/file_util/binary_file_diff.bin",
-      "test/data/file_util/binary_file_same.bin",
-      "test/data/file_util/blank_line.txt",
-      "test/data/file_util/blank_line_crlf.txt",
-      "test/data/file_util/crlf.txt",
-      "test/data/file_util/different.txt",
-      "test/data/file_util/different_first.txt",
-      "test/data/file_util/different_last.txt",
-      "test/data/file_util/empty1.txt",
-      "test/data/file_util/empty2.txt",
-      "test/data/file_util/first1.txt",
-      "test/data/file_util/first2.txt",
-      "test/data/file_util/original.txt",
-      "test/data/file_util/same.txt",
-      "test/data/file_util/same_length.txt",
-      "test/data/file_util/shortened.txt",
-      "test/data/json/bom_feff.json",
-      "test/data/pe_image_reader/signed.exe",
-      "test/data/serializer_nested_test.json",
-      "test/data/serializer_test.json",
-      "test/data/serializer_test_nowhitespace.json",
-    ]
+copy("base_unittests_bundle_data") {
```

Comment:
Do you know how other platforms chromium uses access these files?

If we do need this copy step, I think I would rather maintain a separate (and separately named) target just to be very clear about upstream boundaries

Reply:
I don't know how other platforms chromium uses to access these files, but it's currently not only for `is_apple` in chromium trunk. Maybe it's some weird setting for M114?

And to use copy is for this `$sb_static_contents_output_data_dir` for Starboard as it does not support relative path? What's the benefit for maintain a separate target?